### DISCUSSION TITLE
Post v0.2.0 feature ideas

### DISCUSSION THREAD

**Discussion by andrewdavidmackenzie** [andrewdavidmackenzie/pigg#201] [General]
Soon, we'll start thinking about what features to work on for the next (v0.3.0) release.

Our initial idea is to focus on the networked/remote hardware idea, so you can run the GUI on one machine and view/control a remote Pi.

But we'd love to hear your ideas for new features and improvements to existing ones!

**Comment by arqtv** [comment]
This look like Piggui could be a valuable tool for RPi Makers - many of whom are hardware rather than software savvy so a simple way to install Piggui on a Pi would be a good first move - with easy and complete instructions for installation. That way folk like me can interact with the tool and offer useful hints, tips, bugs and ideas to your project. I can see there is zipped up source code available but no instructions so at a minimum please drop in a wget <URL> for download and an either an install script or simple cli commands to install it. And a remote control over a LAN is a great idea, especially from and to a terminal rather than a Desktop. Happy to do some testing if/when I can install it.

**Reply by andrewdavidmackenzie** [reply]
Understood and agreed. 

We have issues to automate building on pre-built binaries to help with this, plus `cargo binstall` support 

Not sure we will go as far as a real "installer", but avoiding the need to build or cross-build from source should be a big help, then we will re-assess.

**Comment by andrewdavidmackenzie** [comment]
I plan to improve the docs for "building from source" (including the release tarball of source) in this next release.

We had already started the networking feature and it is close to completion, so we will finish and deliver a first version of that in 0.3.0.

Then, next on our list is pre-built binaries and ease of installation (not built from source) which I hope will cover your (and other Maker's) needs.

The order may not be the one you would have preferred, but it shouldn't be too long before we have this work done!

**Comment by andrewdavidmackenzie** [comment]
@arqtv Please see: https://github.com/andrewdavidmackenzie/pigg/pull/266 and it's BUILDING.md file.
I have just tested those instructions downloading and building and running the 0.2.0 release from the source tarball/zip in the release.

If you see any room for clarification or improvement, then please either let me know or send a PR with some proposed changed!

Much appreciated!
